### PR TITLE
Shu ting

### PR DESCRIPTION
**Pull request by LenaHenningsen**
Hi there, my attempt to add Shu Ting / Gao Hua / Gorki to the database.

### CODE REVIEW

**Comment by inamoto4** [comment]
Looks good, thank you! :-)
A few remarks:

1) I would write out the first name in Chinese, too, like 马克西姆 for Maxim (Gorki). 
@duncdrum 
2) I do not know the best way to mark unknown information. Is it by question mark, "unclear" or sth else? 
Would probably relate to the issue with missing information which could later be filled out by our student assistants. For these I would not necessarily write "MIRA", as we still have Yi-ting and Chia-yi. ;-)
3) The place coordinates are missing, they can be found on latlong.net.
4) There are no source information on the persons given. Wikipedia can easily be copied in, yet the information on Shu Ting probably comes from somewhere else, right?
5) I guess "0" is "NULL" now. 
6) Do we include the Russian original titles or not?

**Comment by duncdrum** [comment]
@inamoto4 you can use post comments on the individual lines pull request as part of the review process. You can read about it [here](https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/).

i agree with 1, 3, 4. 
2) depends on the datatype of the field. There is no general rule, as long as you define the difference between an empty field, and the different forms of unknown, *except*
5) ``0`` and ``NULL`` are *never* the same, the former is a value, the later is like ``n.a.``.

**Comment by LenaHenningsen** [comment]
Hi there,
1) sure
2) I would actually like to assign tasks in this way to concrete persons.
Thanks

**Comment by LenaHenningsen** [comment]
For me the characters look ok. Will check again tomorrow.
Best,
Lena

File: data/Place.csv [hunk 1]
```diff
@@ -26,7 +26,7 @@ PL0024,Sheker,42.544292,71.171379,2017-07-09,OS,2017-07-09,OS,
 PL0025,Bolshoy Fontan,46.482526,30.72331,2017-07-09,OS,2017-07-09,OS,
 PL0026,Kazan,55.830431,49.066081,2017-07-09,OS,2017-07-09,OS,
 PL0027,Moscow,55.755826,37.6173,2017-07-09,OS,2017-07-09,OS,
-PL0028,Birmendre�s,36.735349,3.050374,2017-07-09,OS,OS,OS,
```
Comment (inamoto4) [inline data/Place.csv]: zwei i-Punkte. sieht komisch aus

**Comment by duncdrum** [comment]
text.csv needs to be cleaned up.

**Pull request merged**